sass/_chatrooms.scss

```diff
@@ -93,6 +93,9 @@
                 font-size: var(--font-size-small);
             }
         }
+        .show-room-details-modal{
+            cursor: pointer;
+        }
```

Comment:
I'd like us to avoid a proliferation of CSS classes. Can you not just add `cursor: pointer` to  `.show-user-details-modal` directly, instead of wrapping it in a div with a new class?

Also, don't we need to modify the `click` event handler in JavaScript to now listen for `click` on a different DOM element?

Reply:
I deleted the div wrapping the avatar. As for the click event, there is no need to do that because I added .show-user-details-modal and .show-room-details-modal classes to the DOM elements that are meant to showcase groupchat/user info on click, so It's already handled.

Reply:
Ok, looks much cleaner already. One more suggestion, how about putting `show-user-details-modal` on the `div.chatbox-title` element. You're already letting that element show a cursor, so it makes sense to also make it the same element that's clickable.

Reply:
Done, although it actually makes the entire top area with the avatar and the nickname and the area next to it until right before the close button clickable..

Reply:
Sorry but that doesn't sound right... then there shouldn't have been `cursor: pointer` on `.chat-title`.